modules/persistent-kubelet-log-level-configuration.adoc

```diff
@@ -0,0 +1,36 @@
+[id="persistent-kubelet-log-level-configuration_{context}"]
+= Persistent kubelet log level configuration
+
+.Procedure
+
+. Use the following `MachineConfig` for persistent kubelet log level configuration:
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+ kind: MachineConfig
+ metadata:
+   labels:
+     machineconfiguration.openshift.io/role: master
+   name: 99-master-kubelet-loglevel
+ spec:
+   config:
+     ignition:
+       config: {}
+       security:
+         tls: {}
+       timeouts: {}
+       version: 2.2.0
+     systemd:
+       units:
+         - name: kubelet.service
+           enable: true
+           dropins:
+             - name: 10-kubelet-loglevel.conf
+               contents: |
+                 [Service]
+                 Environment=
+                 Environment="KUBELET_LOG_LEVEL=2"
```

Comment:
Need some formatting correction to remove leading space from second line kind: MachineConfig. That is creating syntax error when copying and creating machine config.

Reply:
The name 'kubelet.service' needs to be changes to kubelet-loglevel.service. The dropin name should be named to 20-logging.conf or 30-logging.conf (to override the previous value).

Reply:
> The name 'kubelet.service' needs to be changes to kubelet-loglevel.service. The dropin name should be named 20-logging.conf or 30-logging.conf (to override the previous value).

@rphillips hi, couple of observation here. I see after changing kubelet.service to kubelet-loglevel.service, I see 20-logging.conf file created in /etc/systemd/system/kubelet-loglevel.service.d directory but does not change the loglevel. But keeping kubelet.service and naming the file 30-logging.conf does work. Is this correct behavior?